.github/workflows/python.yaml

```diff
@@ -284,10 +281,38 @@ jobs:
         MODULE: ${{ matrix.module }}
       run: |
         if [[ "$MODULE" == "xorbits" ]]; then
-          pytest --ignore xorbits/_mars/ --ignore xorbits/xgboost --ignore xorbits/lightgbm \
-            --ignore xorbits/datasets --timeout=1500 \
+          pytest --ignore xorbits/_mars/ --ignore xorbits/pandas --ignore xorbits/numpy \
+            --ignore xorbits/xgboost --ignore xorbits/lightgbm --ignore xorbits/sklearn \
+            --ignore xorbits/datasets \
+            --ignore xorbits/core/tests/test_execution_exit.py \
+            --timeout=1500 \
+            -W ignore::PendingDeprecationWarning \
+            --cov-config=setup.cfg --cov-report=xml --cov=xorbits \
+            xorbits
+          # workaround: this case will hang, run it separately.
+          pytest --timeout=1500 \
```

Comment:
Actually, the pytest supports run some function in a file so maybe we don't need to separate it into a single file.

Reply:
I use  `pytest -k "not test_function_name"` to filter out the hang cases. And `-k "test_function_name"` to test hang cases separately.